php/antidot/var/cache/config-cache.php

```diff
@@ -0,0 +1,120 @@
+<?php
```

Comment:
is this file suitable for production ?

Reply:
Is an auto-generated cache, not needed, I will remove it. When I apply al suggested changes I will request another revision, thank you for your help,- D